Graphs/Astar.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Author: Mathang Peddi
```

Comment:
Please use JSDoc annotations like `@author`.

Reply:
Have changed it to @author: Mathang Peddi
Do I need to make any other change here?